samcli/lib/telemetry/event.py

```diff
@@ -48,10 +48,7 @@ class EventType:
     _WORKFLOWS = [f"{config.language}-{config.dependency_manager}" for config in ALL_CONFIGS]
 
     _event_values = {  # Contains allowable values for Events
-        EventName.USED_FEATURE: [
-            "Accelerate",
-            "CDK",
-        ],
+        EventName.USED_FEATURE: ["Accelerate", "CDK", "ApplicationInsights"],
```

Comment:
```suggestion
        EventName.USED_FEATURE: ["Accelerate", "CDK", "InitWithApplicationInsights"],
```

Reply:
Should we move these events names into constants so that they can be used in different places? I saw we are duplicating the string here and somewhere above.

Reply:
I think making UsedFeature an Enum class makes more sense